app/models/miq_region.rb

```diff
@@ -252,6 +252,30 @@ def remote_ws_url
     hostname && URI::HTTPS.build(:host => hostname).to_s
   end
 
+  def remote_cockpit_ws_miq_server
+    MiqServer.in_region(region).find_by(:has_active_cockpit_ws => true)
+  end
+
+  def remote_cockpit_ws_address
+    contact_with = VMDB::Config.new("vmdb").config.fetch_path(:webservices, :contactwith)
```

Comment:
Please use `::Settings` here as `VMDB::Config` is deprecated (see https://github.com/ManageIQ/manageiq/pull/12506/files#diff-470a9664152da41d77ba238f0e8c6465R237 as an example)